runtime/onert/api/python/include/nnfw_api_wrapper.h

```diff
@@ -33,6 +33,31 @@ namespace python
 
 namespace py = pybind11;
 
+/**
+ * @brief Data type mapping between NNFW_TYPE and numpy dtype.
+ */
+struct dtype
+{
+private:
+  NNFW_TYPE _nnfw_type;
+  py::dtype _py_dtype;
+  // The name of the dtype, e.g., "float32", "int32", etc.
+  // This is mainly for the __repr__ implementation.
+  const char *_name;
+
+public:
+  dtype() : dtype(NNFW_TYPE::NNFW_TYPE_TENSOR_FLOAT32) {}
+  explicit dtype(NNFW_TYPE type);
+
+  auto name() const { return _name; }
```

Comment:
I'm not so sure about using `auto` as return types of methods. Not a very common thing in my opinion and not really used in this repository either.

Reply:
Replaced with explicit types.